lib/eventOrNewsFetch.js

```diff
@@ -55,16 +55,16 @@ export function mapAndSanitate(item, type) {
   };
 }
 
-export async function EventFetch(_, args, req) {
+export async function fetchOne(_, args, req, docType = 'event') {
```

Comment:
Don't think we need to the default argument here. A `docType` should be mandatory.